portal-impl/src/com/liferay/portal/service/impl/WorkflowDefinitionLinkLocalServiceImpl.java

```diff
@@ -382,6 +408,41 @@ public WorkflowDefinitionLink updateWorkflowDefinitionLink(
 		return workflowDefinitionLinkPersistence.update(workflowDefinitionLink);
 	}
 
+	@Override
+	public WorkflowDefinitionLink updateWorkflowDefinitionLink(
+			Long groupId, String className,
+			WorkflowDefinitionLink oldWorkflowDefinitionLink)
```

Comment:
We only use the old prefix when we also have another parameter of the same type, for example `(WorkflowDefinitionLink newWorkflowDefinitionLink, WorkflowDefinitionLink oldWorkflowDefinitionLink)`, so please rename this variable to `workflowDefinitionLink`.

To avoid conflicting with the other variable you created inside the method, you can change this other variable from `workflowDefinitionLink` to `existingWorkflowDefinitionLink`. Please see [this](https://github.com/liferay/liferay-portal/blob/12d419627dec40e879aeec074fd38940938ae694/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java#L199-L202) example.